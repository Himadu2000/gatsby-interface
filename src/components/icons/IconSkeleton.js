import React from "react"
import PropTypes from "prop-types"

const ICON_SIZES = {
  xsmall: `xsmall`,
  small: `small`,
  medium: `medium`,
  large: `large`,
}

const iconHeightBySize = {
  [ICON_SIZES.xsmall]: `16px`,
  [ICON_SIZES.small]: `24px`,
  [ICON_SIZES.medium]: `32px`,
  [ICON_SIZES.large]: `40px`,
}

export default function IconSkeleton({
  iconName,
  size = ICON_SIZES.small,
  style,
  applyColorToStroke = true,
  ...rest
}) {
  return (
    <svg
      preserveAspectRatio="xMidYMid meet"
      height={iconHeightBySize[size]}
      width={iconHeightBySize[size]}
      viewBox="0 0 24 24"
      fill={applyColorToStroke ? `none` : `currentColor`}
      stroke={applyColorToStroke ? `currentColor` : `none`}
      strokeWidth="1"
      fillRule="evenodd"
      data-testid={`icon-${iconName}`}
      style={{
        verticalAlign: `middle`,
        ...(style || {}),
      }}
      {...rest}
    />
  )
}

IconSkeleton.propTypes = {
  iconName: PropTypes.string.isRequired,
  size: PropTypes.oneOf(Object.values(ICON_SIZES)),
  applyColorToStroke: PropTypes.bool,
}
