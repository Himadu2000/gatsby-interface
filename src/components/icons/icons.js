export { default as BlogIcon } from "./BlogIcon"
export { default as CheckIcon } from "./CheckIcon"
export { default as PortfolioIcon } from "./PortfolioIcon"
export { default as EcommerceIcon } from "./EcommerceIcon"
