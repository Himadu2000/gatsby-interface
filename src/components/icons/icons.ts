export * from "./types"
export { default as BlogIcon } from "./BlogIcon"
export { default as CheckIcon } from "./CheckIcon"
export { default as PortfolioIcon } from "./PortfolioIcon"
export { default as EcommerceIcon } from "./EcommerceIcon"
export { default as CheckCircleIcon } from "./CheckCircleIcon"
export { default as EllipsisIcon } from "./EllipsisIcon"
export { default as InProgressIcon } from "./InProgressIcon"
export { default as CloseCircleIcon } from "./CloseCircleIcon"
