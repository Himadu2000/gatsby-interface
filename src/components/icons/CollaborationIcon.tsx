import React from "react"
import IconSkeleton from "./IconSkeleton"
import { IconProps } from "./types"
import { useTheme } from "../ThemeProvider"

export default function CollaborationIcon(props: IconProps) {
  const theme = useTheme()
  return (
    <IconSkeleton {...props} iconName="CollaborationIcon">
      <circle
        cx="16"
        cy="6"
        r="1"
        fill={theme.colors.purple[40]}
        stroke={theme.colors.purple[40]}
        strokeWidth="1.5"
      />
      <circle
        cx="9"
        cy="10"
        r="1"
        fill={theme.colors.purple[30]}
        stroke={theme.colors.purple[30]}
        strokeWidth="1.5"
      />
      <circle
        cx="23"
        cy="10"
        r="1"
        fill={theme.colors.blue[50]}
        stroke={theme.colors.blue[50]}
        strokeWidth="1.5"
      />
      <path
        d="M7.75 27.25L5.736 24.733C5.09773 23.9351 4.75 22.9438 4.75 21.922V16.75C4.75 16.3522 4.90804 15.9706 5.18934 15.6893C5.47064 15.408 5.85218 15.25 6.25 15.25V15.25C6.64782 15.25 7.02936 15.408 7.31066 15.6893C7.59196 15.9706 7.75 16.3522 7.75 16.75V20.5"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M12.25 27.114V24.114C12.2504 23.2242 11.987 22.3542 11.493 21.614L10.351 19.9C10.2536 19.7375 10.1242 19.5965 9.97062 19.4855C9.81708 19.3745 9.64262 19.2958 9.45778 19.2542C9.27294 19.2126 9.08159 19.2089 8.8953 19.2435C8.70901 19.278 8.53166 19.3499 8.374 19.455V19.455C8.08936 19.6447 7.884 19.9319 7.79668 20.2626C7.70935 20.5933 7.7461 20.9446 7.9 21.25L9.25 23.5"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M24.25 27.25L26.264 24.733C26.9023 23.9351 27.25 22.9438 27.25 21.922V16.75C27.25 16.3522 27.092 15.9706 26.8107 15.6893C26.5294 15.408 26.1478 15.25 25.75 15.25C25.3522 15.25 24.9706 15.408 24.6893 15.6893C24.408 15.9706 24.25 16.3522 24.25 16.75V20.5"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <path
        d="M19.75 27.114V24.114C19.7496 23.2242 20.013 22.3542 20.507 21.614L21.649 19.9C21.7464 19.7375 21.8758 19.5965 22.0294 19.4855C22.1829 19.3745 22.3574 19.2958 22.5422 19.2542C22.7271 19.2126 22.9184 19.2089 23.1047 19.2435C23.291 19.278 23.4683 19.3499 23.626 19.455C23.9106 19.6447 24.116 19.9319 24.2033 20.2626C24.2907 20.5933 24.2539 20.9446 24.1 21.25L22.75 23.5"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
        strokeLinecap="round"
        strokeLinejoin="round"
      />
      <circle
        cx="16"
        cy="15"
        r="4"
        fill={theme.colors.purple[30]}
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
    </IconSkeleton>
  )
}
