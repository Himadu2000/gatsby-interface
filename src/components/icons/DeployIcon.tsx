import React from "react"
import IconSkeleton from "./IconSkeleton"
import { IconProps } from "./types"
import { useTheme } from "../ThemeProvider"

export default function DeployIcon(props: IconProps) {
  const theme = useTheme()
  return (
    <IconSkeleton {...props} iconName="DeployIcon">
      <path
        d="M11.5 9H20.5C22.433 9 24 10.567 24 12.5V12.5C24 14.433 22.433 16 20.5 16H18"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <path
        d="M18 23L11.5 23C9.567 23 8 21.433 8 19.5V19.5C8 17.567 9.567 16 11.5 16L13.5 16"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <path
        d="M15.0003 20.1716L17.8287 23L15.0003 25.8284"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <circle
        r="2.25"
        transform="matrix(1 0 0 -1 9 9)"
        fill={theme.colors.purple[30]}
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <circle
        r="2.25"
        transform="matrix(1 0 0 -1 16 16)"
        fill={theme.colors.orange[70]}
        stroke={theme.colors.orange[70]}
        strokeWidth="1.5"
      />
      <circle
        r="3"
        transform="matrix(1 0 0 -1 23 23)"
        fill={theme.colors.purple[60]}
      />
    </IconSkeleton>
  )
}
