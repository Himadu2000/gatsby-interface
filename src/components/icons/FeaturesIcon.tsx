import React from "react"
import IconSkeleton from "./IconSkeleton"
import { IconProps } from "./types"
import { useTheme } from "../ThemeProvider"

export default function FeaturesIcon(props: IconProps) {
  const theme = useTheme()

  return (
    <IconSkeleton {...props} iconName="FeaturesIcon">
      <path
        d="M18.9246 14.6967L16.8033 12.5754L13.2678 16.1109L15.3891 18.2322L18.9246 14.6967Z"
        fill={theme.colors.white}
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
        strokeMiterlimit="10"
      />
      <path
        fill-rule="evenodd"
        clip-rule="evenodd"
        d="M23.7071 6.00595L26.8891 4.81801L25.7011 7.99999L26.8891 11.182L23.7071 9.99403L20.5251 11.182L21.7131 7.99999L20.5251 4.81801L23.7071 6.00595Z"
        fill={theme.colors.purple[60]}
      />
      <path
        fillRule="evenodd"
        clipRule="evenodd"
        d="M11.472 8L12 9.41421L10.5858 8.88624L9.17154 9.41421L9.69951 8L9.17154 6.58579L10.5858 7.11376L12 6.58579L11.472 8Z"
        fill={theme.colors.orange[70]}
        stroke={theme.colors.orange[70]}
        strokeWidth="1.5"
      />
      <path
        fillRule="evenodd"
        clipRule="evenodd"
        d="M24.8863 18L25.4143 19.4142L24.0001 18.8862L22.5858 19.4142L23.1138 18L22.5858 16.5858L24.0001 17.1138L25.4143 16.5858L24.8863 18Z"
        fill={theme.colors.orange[70]}
      />
      <path
        d="M13.2678 16.1109L15.3891 18.2322L7.25735 26.3639C6.67157 26.9497 5.72182 26.9497 5.13603 26.3639C4.55025 25.7782 4.55025 24.8284 5.13603 24.2426L13.2678 16.1109Z"
        fill={theme.colors.purple[20]}
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
    </IconSkeleton>
  )
}
