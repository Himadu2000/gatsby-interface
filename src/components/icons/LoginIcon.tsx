import React from "react"
import IconSkeleton from "./IconSkeleton"
import { IconProps } from "./types"
import { useTheme } from "../ThemeProvider"

export default function LoginIcon(props: IconProps) {
  const theme = useTheme()
  return (
    <IconSkeleton {...props} iconName="LoginIcon">
      <path
        d="M23.5882 8.89494C25.3031 15.2949 21.5051 21.8734 15.1051 23.5882C8.70505 25.3031 2.12664 21.5051 0.411766 15.1051C-1.30311 8.70505 2.49494 2.12664 8.89494 0.411766C15.2949 -1.30311 21.8734 2.49494 23.5882 8.89494Z"
        fill="#FFFAFA"
      />
      <path
        d="M19.3873 10.0206C20.4805 14.1005 18.0593 18.2941 13.9794 19.3873C9.89951 20.4805 5.70588 18.0593 4.61267 13.9794C3.51946 9.89953 5.94066 5.70589 10.0206 4.61269C14.1005 3.51948 18.2941 5.94068 19.3873 10.0206Z"
        fill="white"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <path
        d="M16.6353 10.7579C17.3212 13.3179 15.802 15.9493 13.242 16.6352C10.682 17.3212 8.05064 15.8019 7.36469 13.2419C6.67874 10.6819 8.19796 8.05058 10.758 7.36463C13.318 6.67868 15.9493 8.1979 16.6353 10.7579Z"
        fill={theme.colors.purple[60]}
      />
      <path
        d="M14.1311 1.53688C14.5548 0.871151 14.0765 -4.51356e-08 13.2875 -7.96279e-08L10.8217 -1.87409e-07C10.0326 -2.21901e-07 9.55441 0.87115 9.97805 1.53688L11.2109 3.47425C11.6039 4.09174 12.5053 4.09175 12.8982 3.47425L14.1311 1.53688Z"
        fill={theme.colors.red[50]}
      />
      <path
        d="M1.59153 9.92351C0.925808 9.49986 0.054657 9.97808 0.054657 10.7672L0.054657 13.2329C0.054657 14.022 0.925808 14.5002 1.59153 14.0766L3.5289 12.8437C4.1464 12.4507 4.1464 11.5493 3.5289 11.1564L1.59153 9.92351Z"
        fill={theme.colors.red[50]}
      />
      <path
        d="M9.97809 22.4631C9.55445 23.1288 10.0327 24 10.8217 24L13.2875 24C14.0766 24 14.5548 23.1288 14.1312 22.4631L12.8983 20.5258C12.5053 19.9083 11.6039 19.9083 11.211 20.5258L9.97809 22.4631Z"
        fill={theme.colors.red[70]}
      />
      <path
        d="M22.5178 14.0765C23.1835 14.5002 24.0546 14.022 24.0546 13.2329L24.0546 12.2944C24.0546 11.5053 23.1835 11.0271 22.5178 11.4507L21.7804 11.92C21.1629 12.3129 21.1629 13.2143 21.7804 13.6073L22.5178 14.0765Z"
        fill={theme.colors.orange[70]}
      />
    </IconSkeleton>
  )
}
