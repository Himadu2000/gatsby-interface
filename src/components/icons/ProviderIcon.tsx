import React from "react"
import IconSkeleton from "./IconSkeleton"
import { IconProps } from "./types"
import { useTheme } from "../ThemeProvider"

export default function ProviderIcon(props: IconProps) {
  const theme = useTheme()
  return (
    <IconSkeleton {...props} iconName="ProviderIcon">
      <line
        x1="17.25"
        y1="18"
        x2="17.25"
        y2="22"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <line
        x1="20.0554"
        y1="10.8839"
        x2="23.591"
        y2="7.34838"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <line
        x1="10.7358"
        y1="9.59828"
        x2="14.8316"
        y2="12.4662"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <line
        x1="19.7358"
        y1="15.5983"
        x2="23.8316"
        y2="18.4662"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <line
        x1="7.59095"
        y1="23.3484"
        x2="14.5909"
        y2="16.3484"
        stroke={theme.colors.blue[50]}
        strokeWidth="1.5"
      />
      <circle
        cx="17"
        cy="14"
        r="4"
        fill={theme.colors.purple[30]}
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <circle
        cx="25"
        cy="19"
        r="2"
        fill={theme.colors.purple[60]}
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <circle
        cx="9"
        cy="8.5"
        r="2"
        fill="white"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <circle
        cx="17.25"
        cy="24"
        r="2"
        fill="white"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
      <circle
        cx="6"
        cy="25"
        r="2"
        fill={theme.colors.blue[50]}
        stroke={theme.colors.blue[50]}
        strokeWidth="1.5"
      />
      <circle
        cx="24"
        cy="7"
        r="2"
        fill="white"
        stroke={theme.colors.purple[60]}
        strokeWidth="1.5"
      />
    </IconSkeleton>
  )
}
