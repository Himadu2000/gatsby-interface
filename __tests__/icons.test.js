import React from "react"
import { render } from "react-testing-library"

import * as icons from "../src/components/icons/icons"

const sortedIconComponentNames = Object.keys(icons).sort()

describe(`icons`, () => {
  sortedIconComponentNames
    // filter out __esModule
    .filter(componentName => typeof icons[componentName] !== `boolean`)
    .forEach(componentName => {
      describe(`<${componentName} />`, () => {
        const IconComponent = icons[componentName]

        it(`renders unchanged`, () => {
          const { container } = render(<IconComponent />)

          expect(container).toMatchSnapshot()
        })
      })
    })
})
