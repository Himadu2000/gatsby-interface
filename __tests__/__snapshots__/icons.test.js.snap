// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icons <BlogIcon /> renders unchanged 1`] = `
<div>
  <svg
    data-testid="icon-blog"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="currentColor"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      d="M8.49188 11.265L7.43188 15.508L11.6739 14.447L18.0379 8.083L14.8559 4.901L8.49188 11.265Z"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M21.751 4.37097L22.811 5.43097C22.9504 5.57027 23.061 5.73567 23.1364 5.91772C23.2119 6.09978 23.2507 6.29491 23.2507 6.49197C23.2507 6.68904 23.2119 6.88417 23.1364 7.06622C23.061 7.24827 22.9504 7.41367 22.811 7.55297L19.5 10.863"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M18.038 8.083L21.751 4.371C22.0322 4.0897 22.1901 3.70824 22.1901 3.3105C22.1901 2.91275 22.0322 2.53129 21.751 2.25L20.69 1.189C20.4087 0.90779 20.0272 0.749817 19.6295 0.749817C19.2317 0.749817 18.8502 0.90779 18.569 1.189L14.856 4.9"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M18.75 14.25V21.75C18.75 22.1478 18.592 22.5294 18.3107 22.8107C18.0294 23.092 17.6478 23.25 17.25 23.25H2.25C1.85218 23.25 1.47064 23.092 1.18934 22.8107C0.908035 22.5294 0.75 22.1478 0.75 21.75V6.75C0.75 6.35218 0.908035 5.97064 1.18934 5.68934C1.47064 5.40804 1.85218 5.25 2.25 5.25H9.75"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</div>
`;

exports[`icons <CheckIcon /> renders unchanged 1`] = `
<div>
  <svg
    data-testid="icon-check"
    fill="currentColor"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="none"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      d="M9.3 16.7l-3.1-4.3A1 1 0 0 1 7.8 11l3.7 3.9c.6.6.6 1.6 0 2.1-.7.5-1.7.4-2.2-.3z"
    />
    <path
      d="M9.6 17c-.7-.6-.7-1.5-.2-2.2l7.3-8a1 1 0 0 1 1.6 1.3l-6.6 8.6c-.5.7-1.5.8-2.1.3z"
    />
    <path
      d="M9.7 13c.4.5.7.8 1.1.3.5-.5-.1 1.3-.1 1.3l-1.5-.5.5-1z"
    />
  </svg>
</div>
`;

exports[`icons <EcommerceIcon /> renders unchanged 1`] = `
<div>
  <svg
    data-testid="icon-ecommerce"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="currentColor"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      d="M3.75 5.25001C3.74961 4.96756 3.80495 4.6878 3.91286 4.42677C4.02076 4.16574 4.17912 3.92857 4.37884 3.72885C4.57857 3.52912 4.81574 3.37077 5.07677 3.26286C5.33779 3.15496 5.61755 3.09961 5.9 3.10001"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M0.75 4.687C0.75 3.64284 1.16479 2.64145 1.90312 1.90312C2.64145 1.16479 3.64284 0.75 4.687 0.75"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M10.5001 16.5H7.6731C7.39125 16.5008 7.11717 16.4077 6.89422 16.2353C6.67127 16.0628 6.51217 15.821 6.4421 15.548L4.5421 9.061C4.49636 8.87508 4.49359 8.68119 4.534 8.49405C4.57441 8.3069 4.65694 8.13143 4.77532 7.98095C4.89369 7.83047 5.04481 7.70895 5.21718 7.62561C5.38955 7.54228 5.57864 7.49932 5.7701 7.5H18.9801C19.1719 7.49866 19.3614 7.54116 19.5343 7.62424C19.7071 7.70732 19.8587 7.82878 19.9775 7.97937C20.0962 8.12996 20.179 8.30569 20.2195 8.49314C20.26 8.68059 20.2571 8.87482 20.2111 9.061L19.9001 10.116"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M16.4999 7.5L15.6709 3.75"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M9 7.5L9.828 3.75"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M23.135 23.25L23.235 22.128C23.301 21.4017 23.1001 20.6762 22.67 20.0873C22.2399 19.4983 21.6099 19.0862 20.898 18.928L16.726 18V12C16.726 11.6022 16.5679 11.2206 16.2866 10.9393C16.0053 10.658 15.6238 10.5 15.226 10.5C14.8282 10.5 14.4466 10.658 14.1653 10.9393C13.884 11.2206 13.726 11.6022 13.726 12V21.75L12.026 20.472C11.731 20.2504 11.3659 20.1428 10.9979 20.1688C10.6298 20.1949 10.2836 20.3528 10.0227 20.6137C9.7618 20.8746 9.60383 21.2209 9.5778 21.5889C9.55176 21.9569 9.65942 22.322 9.88098 22.617L10.355 23.25"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</div>
`;

exports[`icons <PortfolioIcon /> renders unchanged 1`] = `
<div>
  <svg
    data-testid="icon-portfolio"
    fill="none"
    fill-rule="evenodd"
    height="24px"
    preserveAspectRatio="xMidYMid meet"
    stroke="currentColor"
    stroke-width="1"
    style="vertical-align: middle;"
    viewBox="0 0 24 24"
    width="24px"
  >
    <path
      d="M6.43308 15.749C6.06531 14.8383 5.92674 13.8512 6.02954 12.8744C6.13233 11.8976 6.47335 10.961 7.02265 10.1468C7.57196 9.33253 8.31275 8.66561 9.17999 8.20454C10.0472 7.74347 11.0144 7.50235 11.9966 7.50235C12.9788 7.50235 13.9459 7.74347 14.8132 8.20454C15.6804 8.66561 16.4212 9.33253 16.9705 10.1468C17.5198 10.961 17.8608 11.8976 17.9636 12.8744C18.0664 13.8512 17.9279 14.8383 17.5601 15.749"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M10.1551 7.78601L11.8461 10.048C11.9643 10.2056 12.0503 10.3849 12.0992 10.5757C12.1481 10.7666 12.1589 10.9651 12.131 11.1601C12.1032 11.3551 12.0372 11.5428 11.9368 11.7123C11.8364 11.8817 11.7037 12.0298 11.5461 12.148L9.74609 13.498V14.248C9.74609 14.6458 9.58806 15.0274 9.30675 15.3087C9.02545 15.59 8.64392 15.748 8.24609 15.748"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M17.629 11.427L15.323 12.587C15.0175 12.7396 14.7732 12.9918 14.6305 13.302C14.4878 13.6121 14.4551 13.9618 14.538 14.293L14.91 15.749"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M6.74609 2.24799V0.747986"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M17.2461 2.24799V0.747986"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M7.5 18.748C7.5 19.1458 7.65804 19.5273 7.93934 19.8086C8.22064 20.09 8.60218 20.248 9 20.248H15C15.3978 20.248 15.7794 20.09 16.0607 19.8086C16.342 19.5273 16.5 19.1458 16.5 18.748"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M22.2461 15.748H1.74609C1.19381 15.748 0.746094 16.1957 0.746094 16.748V22.248C0.746094 22.8003 1.19381 23.248 1.74609 23.248H22.2461C22.7984 23.248 23.2461 22.8003 23.2461 22.248V16.748C23.2461 16.1957 22.7984 15.748 22.2461 15.748Z"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
    <path
      d="M3 15.748V3.74799C3 3.35016 3.15804 2.96863 3.43934 2.68733C3.72064 2.40602 4.10218 2.24799 4.5 2.24799H19.5C19.8978 2.24799 20.2794 2.40602 20.5607 2.68733C20.842 2.96863 21 3.35016 21 3.74799V15.748"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</div>
`;
